### PR TITLE
Docs: simplify build doc

### PR DESCRIPTION
**Pull request by daixiang0**
Signed-off-by: Xiang Dai <long0dai@foxmail.com>

We should better hide details even for contributor and provide a simple way to them.

**What type of PR is this?**
/kind documentation

### CODE REVIEW

**Review by kevin-wangzefeng — COMMENTED** [review]
/approve

File: docs/setup/build.md [hunk 1]
```diff
@@ -1,60 +1,33 @@
-# Cross Compiling KubeEdge
+# Build from source
 
-In most of the cases, when you are trying to compile KubeEdge edgecore on Raspberry Pi or any other device, you may run out of memory, in that case, it is advisable to cross-compile the Edgecore binary and transfer it to your edge device.
```
Comment (fisherxu) [inline docs/setup/build.md:3]: `out of memory` is needed here for fresh developers.

Comment (daixiang0) [reply docs/setup/build.md:3]: Done.

**Pull request merged**